content/analyze/portals/visualization-portals.md

```diff
@@ -15,4 +15,17 @@ linked:
     - ./cellxgene.md
 ---
 
-## Visualization Portals
+
+## Analysis Portals
+
+
+The HCA Analysis Portal Registry lists portals that provide a human-friendly UI in a web or native app to search, explore and analyze biological data.  
+
+These solutions are not built by the HCA DCP implementation team but are solutions we foster and support.   
```

Comment:
A bit shorter and less redundant, while preserving Tim's original intent to distinguish:

```suggestion
These solutions are built by third parties, and fostered by the HCA DCP.
```